.gitignore

```diff
@@ -250,3 +357,7 @@ ModelManifest.xml
 # FAKE - F# Make
 .fake/
 /WebCrawler/WebCrawler/__pycache__
+
+
+## Files created by IntelliJ.
+.idea/
```

Comment:
There should also be a files created by visual studio code. If you want me to I can add this.

Reply:
Should still be in this. Under the section "Ignore Visual Studio temporary files"